yesod-form/Yesod/Form/I18n/Romanian.hs

```diff
@@ -0,0 +1,31 @@
+{-# LANGUAGE OverloadedStrings #-}
+module Yesod.Form.I18n.Romanian where
+
+import Yesod.Form.Types (FormMessage (..))
+import Data.Monoid (mappend)
+import Data.Text (Text)
+
+-- |
+-- Romanian translation
+--
+-- @since 1.7.5
+englishFormMessage :: FormMessage -> Text
```

Comment:
I think this ought to be Romanian, rather than English.

```suggestion
romanianFormMessage :: FormMessage -> Text
```

Also, perhaps it's worth writing this with LambdaCase (though I recognise none of the other modules do this).

Reply:
Changed function name from ```englishFormMessage``` to ```romanianFormMessage``` in module ```Yesod.Form.I18n.Romanian```  (fdfac8de6f270f2eefaf44b90cae766bbca3c936).

```LambdaCase``` extension is supported since ```7.6.1```.  Can I still use it? I'm happy to change if that's not a problem.

Reply:
Good point about less extensive support. It's only syntax sugar anyway, so it's not worth the compromise IMO.